client/src/components/FilterPanel.vue

```diff
@@ -452,6 +464,23 @@ const onClear = () => {
   tillDate.value = null;
 };
 
+const onShareButtonClicked = async () => {
+  const { showSuccessAlert } = useSwal();
+
+  let baseUrl = window.location.href;
+  baseUrl =
+    baseUrl.charAt(baseUrl.length - 1) == "/" ? baseUrl.slice(0, -1) : baseUrl;
+  const filterString = Object.entries(activeFilters.value)
+    .map((f) => f[0] + "=" + f[1])
+    .join("&");
+  const filterUrl = baseUrl + "?" + filterString;
+
+  // This works only on https sites???
+  navigator.clipboard.writeText(filterUrl);
```

Comment:
Coole Idee, allerdings erfährt der User nicht, dass der Link jetzt in der Zwischenablage ist. (Wenn der Browser/OS überhaupt erlaubt.)

Ich schaue morgen mal, ob wir nicht auf mobiles die native share Methode benutzen können. 
Auf dem Desktop ein toast, denn da möchte man eigentlich nichts bestätigen müssen. Und dass der Link angezeigt wird bringt auch nicht. Den manuell markieren ist nicht so geile UX.

Reply:
Das der nur in die Zwischenablage kommt, war auch meine erste Idee.
Aber wie es da auch als Kommentar steht. Anscheinend soll das nur auf https Seiten funktionieren und die habe ich im Dev nicht.
Deshalb habe ich es erstmal auch als Link ergänzt. Der Toast ist defintiv die bessere Lösung.